engine/engine_test.go

```diff
@@ -832,12 +834,12 @@ var warningErrors = []error{
 func getPairFromPairs(t *testing.T, p currency.Pairs) (currency.Pair, error) {
 	t.Helper()
 	for i := range p {
-		if p[i].Base.Equal(currency.BTC) {
+		if p[i].Base.Equal(currency.ETH) {
 			return p[i], nil
 		}
 	}
 	for i := range p {
-		if p[i].Base.Equal(currency.ETH) {
+		if p[i].Base.Equal(currency.BTC) {
 			return p[i], nil
 		}
 	}
```

Comment:
You can do an `||` statement instead of iterating over the same slice again. 
```go 
if p[i].Base.Equal(currency.ETH) || p[i].Base.Equal(currency.BTC)
```

Reply:
Main reason was to prioritise ETH over BTC

Reply:
oh that makes sense now. Will resolve.